grapher/controls/CollapsibleList.sampleInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
```

Comment:
File naming and organization is great.

Should we put these in one folder? Either nested in Controls/ or at the same depth?

Not just for this one, but all the components. As at first you have a component, and then a stylesheet, and then a test file, and then a story file, etc.

Reply:
done